src/metanetx/data.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2019, Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Data classes for MetaNetX data."""
```

Comment:
I'm a bit confused that you originally said that you didn't want to create (SQL) data models but now you have created data model classes anyway.

Reply:
I proposed keeping the data in memory for the lifetime of the service as opposed to creating and administering a SQL database for it. It has little to do with data classes per se.